dracut/image-boot/eos-image-boot-setup

```diff
@@ -18,9 +18,27 @@ if getargbool 0 endless.live_boot; then
   kpartx_roflag=-r
 fi
 
+fstype=$(lsblk -o FSTYPE "${host_device}")
```

Comment:
By default, `lsblk` includes column headings:

```
% lsblk -o FSTYPE /dev/sda1
FSTYPE
vfat
```

so I think we need `--noheadings` here:

```
% lsblk -o FSTYPE --noheadings /dev/sda1
vfat
```